index.js

```diff
@@ -69,6 +69,23 @@ class Policies {
       ctx.redirect(`${ctx.state.l(this.config.verifyRoute)}${redirect}`);
     else ctx.redirect(`${this.config.verifyRoute}${redirect}`);
   }
+  
+  async ensureOtp (ctx, next) => {
+    if (!ctx.isAuthenticated() || (ctx.state.user.two_factor_enabled && !ctx.session.otp)) {
```

Comment:
I think we need to change this to:

```diff
-    if (!ctx.isAuthenticated() || (ctx.state.user.two_factor_enabled && !ctx.session.otp)) {
+    if (!ctx.isAuthenticated() || (ctx.state.user[this.config.userFields.twoFactorEnabled] && !ctx.session.otp)) {
```

And then you also would have to adjust the passport helper here <https://github.com/ladjs/lad.sh/blob/477d285c3deb1038800b35c7167f4069cd2f3b58/helpers/passport.js#L6 to be:

```diff
-const passport = new Passport(Users, config.passport);
+const passport = new Passport(Users, {
+  userFields: config.userFields,
+  ...config.passport
+});
```

And lastly add a line below here <https://github.com/ladjs/passport/blob/0044f9b58d698afbd5f60aa65f8b21e15b30a702/index.js#L72>:

```diff
    fields: {
      displayName: 'display_name',
      givenName: 'given_name',
      familyName: 'family_name',
      avatarURL: 'avatar_url',
      googleProfileID: 'google_profile_id',
      googleAccessToken: 'google_access_token',
      googleRefreshToken: 'google_refresh_token',
      githubProfileID: 'github_profile_id',
      githubAccessToken: 'github_access_token',
      githubRefreshToken: 'github_refresh_token'
-    }
+    },
+    userFields: {
+      twoFactorEnabled: 'two_factor_enabled'
+    }
  });
```